metadata.json

```diff
@@ -146,6 +146,15 @@
     },
     {
       "operatingsystem": "Gentoo"
+    },
+    {
+      "operatingsystem": "windows",
+      "operatingsystemrelease": [
+        "2008R2",
```

Comment:
Not a requirement to get this merged, but: Could you have a look at https://github.com/camptocamp/facterdb/tree/master/facts and maybe contribute some facts from windows machines? This repo basically has copies from the `facter -j` output. This is used in our tests to mock facts. It would really help us to get some more windows facts.